### DISCUSSION TITLE
Welcome to eza Discussions!

### DISCUSSION THREAD

**Discussion by cafkafk** [eza-community/eza#89] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shameble** [comment]
Sorry I am only a dumb user of eza. Emphazise "dumb" :-) 
So we can only expect some more or less intellegent questions. ... and that's it.
Cheers 
shameble

**Comment by shameble** [comment]
Question:
Installed eza via "cargo install eza"
Routinely updated installed packages via "cargo install-update -a"
Result: 
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "package eza not found"'

What can I do? Where can I search? eza is the first (and up to now only) package with this behaviour.
Cheers
shameble

**Reply by cafkafk** [reply]
I can't reproduce this sadly, and I have literally no idea why, hopefully someone else does :woman_shrugging:
